### PR TITLE
Get upstream user when possible and cerberus var

### PR DESCRIPTION
**Pull request by paigerube14**
Want to get upstream user to properly give access to user in google sheets from loaded-upgrades

Add in new variable for cerberus to ignore pods with certain names

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@paigerube14 Looks good to me.  Let's add this to more jobs, I'll take care of router-perf.

**Comment by qiliRedHat** [comment]
@nathan-weinberg Please add your comments or lgtm. Then I will add lgtm label.

File: Jenkinsfile [hunk 1]
```diff
@@ -203,6 +221,7 @@ pipeline {
                     cp $GSHEET_KEY_LOCATION $WORKSPACE/.gsheet.json
                     export GSHEET_KEY_LOCATION=$WORKSPACE/.gsheet.json
                     export EMAIL_ID_FOR_RESULTS_SHEET=$EMAIL_ID_FOR_RESULTS_SHEET
+
```
Comment (nathan-weinberg) [inline Jenkinsfile:224]: Why the new empty line here?

Comment (paigerube14) [reply Jenkinsfile:224]: I'm not sure.

**Pull request merged**